JS/Lesson13/saffet/script.js

```diff
@@ -3,71 +3,64 @@
 // Declare a variable z and initialize it to the sum of x and y.
 // var, const, let
 const x = 5;
-const y = 3;
+const y = 2;
 const z = x + y;
 console.log('Exercise 1:', z);
-
 console.log('------------------------');
-const name = "onur";
-const age = 32;
-const phrase = "Hello world!";
-const myAge = `${phrase} My name is ${name}. I'm ${age} years old.`;
-const myAge2 = phrase + " My name is " + name + ". I'm " + age + " years old.";
-
-console.log('Exercise 2:',myAge);
-console.log('Exercise 2.2:',myAge2);
 
 // ---
 // 2. Declare a variable name and initialize it to your name as a string.
 // Declare a variable age and initialize it to your age as a number.
 // Declare a variable phrase and initialize it to the string "Hello, world!".
 // Add name and age to the sentence above. "Hello, world! My name is (name). I'm (age) years old.".
-
+var name = "Saffet";
+var age ="26";
+const phrase = "Hello, world!";
+const sentence = phrase + " My name is " + name + "." + " I'm " + age + " years old.";
+console.log('Exercise 2.1:',sentence);
+const sentence2 = `${phrase} My name is ${name}. I'm ${age} years old.`;
+console.log('Exercise 2.2:',sentence2);
 console.log('------------------------');
+
 // ---
 // 3. Declare a variable number1 and initialize it to 10.
 // Declare a variable number2 and initialize it to 3.
 // Declare a variable result and initialize it to the quotient of number1 divided by number2.
 // Declare a variable number3 and initialize it to the difference between number1 and number2.
-const number1 = 10;
-const number2 = 3;
+var number1 = 10;
```

Comment:
It's better to use const or let in modern JS. Var keyword was the only keyword for defining variables in the past and it's still present in the language but you should be able to only use const and let.